find_unicode_control.py

```diff
@@ -62,7 +79,7 @@ def analyze_text_detailed(filename, text, disallowed, msg):
     warned = False
     for t in text:
         line = line + 1
-        subset = [c for c in t if c in disallowed]
+        subset = [c for c in t if chr(ord(c)) in disallowed]
```

Comment:
This should be `_chr(ord(c))`; sorry that's a bug I had introduced :)

Reply:
done